HlibPolishchuk/HW7/homework3.py

```diff
@@ -0,0 +1,15 @@
+words = list("hello")
+
```

Comment:
another way, more optimized:
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result